internal/core/src/query/visitors/ExecPlanNodeVisitor.cpp

```diff
@@ -165,6 +172,22 @@ ExecPlanNodeVisitor::VectorVisitorImpl(VectorPlanNode& node) {
 
     std::unique_ptr<BitsetType> bitset_holder;
     if (node.filter_plannode_.has_value()) {
+        if (node.search_info_.materialized_view_involved) {
+            knowhere::MaterializedViewSearchInfo materialized_view_search_info;
+            const auto expr_info =
+                GatherInfoBasedOnExpr(node.filter_plannode_.value());
```

Comment:
As upper said:
ExprInfo  expr_info;
GatherInfoBaseOnExpr(node.filter_plannode_.value(), expr_info) is better, avoid copy

Reply:
ditto